scss/_settings.scss

```diff
@@ -12,6 +12,7 @@
 @import 'settings_assets';
 @import 'settings_breakpoints';
 @import 'settings_colors';
+@import 'settings_colors_deprecated';
```

Comment:
I'd suggest importing deprecated colors in `settings_colors.scss`, so that anything that imports colours gets all of them automatically.